packages/salesforcedx-vscode-core/src/commands/commands.ts

```diff
@@ -331,12 +331,29 @@ export abstract class SfdxCommandletExecutor<T>
       cwd: getRootWorkspacePath()
```

Comment:
You'll need to add `env: { SFDX_JSON_TO_STDOUT: 'true' }` here for the changes in line 335 to work in all scenarios.